datasets.html

```diff
@@ -1,6 +1,8 @@
 ---
 title: Datasets
 layout: default
+permalink: /datasets/
+activeLink: /datasets/
```

Comment:
There may be a way to set the active nav link by a whole path, BUT it can also be set here at the page level if needed.